src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/GetAllocatedCaseServiceTest.kt

```diff
@@ -108,6 +109,40 @@ class GetAllocatedCaseServiceTest {
     assert(result.activeEvents.first().offences.first().mainOffence == true)
   }
 
+  @Test
+  fun `should not populate assesment date if not assessment found`() = runBlocking {
+    val crn = "X123456"
+
+    val now = LocalDateTime.now()
+    val assessment = Assessment(now, "", "")
```

Comment:
Doesn't look like variable assessment is being used in this test, can it be removed?